### ISSUE TITLE
Do you have a plan to release Maya 2019?

### ISSUE THREAD

**Issue by hirtanak** [Azure/azure-batch-maya#43]
v0.23 does not support yet..

**Comment by davidkydd** [comment]
Yes we do but this has not yet been completed - apologies this has taken a lot longer to release due to difficulties enabling some of the renderers for Maya 2019. Arnold on Maya2019 is supported in the parallel offering BatchExplorer: https://github.com/Azure/BatchExplorer
